interface/wx/bmpbndl.h

```diff
@@ -315,6 +315,9 @@ class wxBitmapBundle
         cached, avoid calling it for many different sizes if you do use it, as
         this will create many bitmaps that will never be deleted and will
         consume resources until the application termination.
+
+        @param size The size of the bitmap to return, in logical pixels. If
+            this parameter is wxDefaultSize, default bundle size is used.
```

Comment:
I'm confused, logical pixels ? I thought we agreed that on this level these are physical pixels, that a bitmap is retrieved here based in true pixel value, at least IIRC that route will be taken with all `GetBitmapFor(wxWindow*) `routing through `GetPreferredSizeAtScale`, maintaining that this is corresponds to `GetSize` on the returned `wxBitmap`, not `GetLogicalSize` ...

Reply:
Sorry, it's me who was confused (of course). I'll fix this, thanks for noticing it!